test/integration/resource.ts

```diff
@@ -0,0 +1,41 @@
+import {expect} from 'chai';
+import {Ketting, NeverCache} from '../../src';
+
+describe('Using resources', async () => {
+  const ketting = new Ketting('http://localhost:3000/hal1.json');
+
+  it('concurrent get on the same resource should do 2 distinct HTTP calls if request headers are different', async () => {
```

Comment:
The string set in `describe` must be the begining of a sentence ended by the string set in `it`.
Please review all your test names.

Reply:
Done